### ISSUE TITLE
Finetune with normal data

### ISSUE THREAD

**Issue by xytmhy** [zqhang/AnomalyCLIP#77]
Could the model be fine-tuned with only normal data? Or I have to use the anomaly data with label mask?
Background: The pretrained model perform bad on my own dataset, and I want to finetune it. But I have no label for my dataset. Thanks.

**Comment by zqhang** [comment]
I am afraid that AnomalyCLIP cannot be fine-tuned with only normal data. You could synthesize pseudo anomalies from your own data for fine-tuning.

**Comment by xytmhy** [comment]
@zqhang Thanks for your comments. You may close the issue then.
